### PR TITLE
data: waellerbe.json

### PR DESCRIPTION
**Pull request by waellerbe**
Original version for my BioDrop JSON file

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by waellerbe** [comment]
> JSON check / check (pull_request)
- I see that I may have a mistake in my JSON file

File: data/waellerbe.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "William A Ellerbe",
+  "bio": "Prompt Engineering Consultant with a passion for Open Source projects"
+}
```
Comment (SaraJaoude) [inline data/waellerbe.json:4]:
```suggestion
  "bio": "Prompt Engineering Consultant with a passion for Open Source projects",
  "links": [
```

**Review by SaraJaoude — APPROVED** [review]
I have fixed the JSON so that it would pass.  

In future you may want to have a look at [Playground](https://www.biodrop.io/playground) so that you can check your JSON in advance. 

If you wish to add more links or any other features, once you are ready please do check our [Documentation](https://www.biodrop.io/docs).  You can choose whether any edits/additions are made through JSON or our Forms.

I will merge your PR now and would encourage you to check out the Create & Manage Profile Dashboard on the BioDrop app so that you can get an idea of how you can customise it further.

**Comment by waellerbe** [comment]
> I have fixed the JSON so that it would pass.
> 
> In future you may want to have a look at [Playground](https://www.biodrop.io/playground) so that you can check your JSON in advance.
> 
> If you wish to add more links or any other features, once you are ready please do check our [Documentation](https://www.biodrop.io/docs). You can choose whether any edits/additions are made through JSON or our Forms.
> 
> I will merge your PR now and would encourage you to check out the Create & Manage Profile Dashboard on the BioDrop app so that you can get an idea of how you can customize it further.

@SaraJaoude : Thank you for your assistance with the JSON file. Per your advice, I will definitely refer to the Playground and the BioDrop documentation. Looking forward to learn how to add changes to my BioDrop profile with the wide range of examples.

**Pull request merged**